modules/grpc/clickhouse/clickhouse-dest.cpp

```diff
@@ -52,6 +52,8 @@ DestDriver::init()
       return false;
     }
 
+  this->query = "INSERT INTO " + this->database + "." + this->table + " FORMAT Protobuf";
```

Comment:
We might need escaping here.

Reply:
Note: I checked and unfortunately the ClickHouse gRPC server does not support query params with the provided proto, so we cannot use this: https://clickhouse.com/docs/en/sql-reference/syntax#defining-and-using-query-parameters

The best we can do is escaping with backtick or double quote: https://clickhouse.com/docs/en/sql-reference/syntax#identifiers

Additionally we could move the database name from the query to the request's database field, so we only escape the table part.

Reply:
FYI: https://github.com/ClickHouse/ClickHouse/pull/71080

Reply:
Done.